scripts/generate_version_docs.py

```diff
@@ -95,18 +95,39 @@ def get_last_updated(image_config):
 
   return terra_date
 
-def get_static_legacy_doc():
-  doc = {
-    "id": 'leonardo-jupyter-dev',
-    "label": 'Legacy (default prior to January 14)',
-    "version": 'FINAL',
-    "updated": '2019-08-26',
-    "packages": 'https://storage.googleapis.com/terra-docker-image-documentation/leonardo-jupyter-dev-versions.json',
-    "image": 'us.gcr.io/broad-dsp-gcr-public/leonardo-jupyter:5c51ce6935da',
-    "requiresSpark": True
-  }
+# See definitions in https://docs.google.com/document/d/1qAp1wJTEx1UNtZ4vz1aV4PZRfjyYfF7QkwwOjK4LoD8/edit
+def get_static_docs():
+  docs = [
+    {
+      "id": 'leonardo-jupyter-dev',
+      "label": 'Legacy Python/R (default prior to January 14, 2020)',
```

Comment:
I wonder at this point, can we remove this image from the list now 🤔

Reply:
🤷‍♂️ we could see if any runtimes are still using it.

Reply:
seems there're still new runtimes created with this image in May...but I still wonder if we should make the decision to nudge people to newer images, and if they really want this image, they can use custom image option.